walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -200,7 +201,7 @@ public void doubleBookedPeople() {
   }
 
   @Test
-  public void justEnoughRoom() {
+  public void query_justEnoughRoom() {
```

Comment:
I am not sure what justEnoughRoom means. Can you use a more accurate wording? Maybe it should be renamed as: query_oneTimeSlotAvailable

Reply:
I used your suggestion.
Done!